### ISSUE TITLE
[Grid][Accessibility] Expand/collapse column header does not have text

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#2255]
The expand/collapse column, related to the Grid master-detail functionality has an empty header (TH) cell which triggers a WAVE accessibility tool error:

![image](https://user-images.githubusercontent.com/7533599/57434731-558a9980-7244-11e9-9b84-2eb353638dce.png)

**Comment by tsvetomir** [comment]
Fixed in the latest version, [see example](https://www.telerik.com/kendo-angular-ui/components/grid/advanced-features/hierarchy/).
